packages/broker/src/plugins/operator/VoteOnSuspectNodeHelper.ts

```diff
@@ -0,0 +1,54 @@
+import { Contract } from '@ethersproject/contracts'
+import { Provider } from '@ethersproject/providers'
+import { Signer } from '@ethersproject/abstract-signer'
+import { operatorABI } from '@streamr/network-contracts'
+import type { Operator } from '@streamr/network-contracts'
+import { Logger } from '@streamr/utils'
+import { OperatorServiceConfig } from './OperatorPlugin'
+
+export const VOTE_KICK = '0x0000000000000000000000000000000000000000000000000000000000000001'
+export const VOTE_NO_KICK = '0x0000000000000000000000000000000000000000000000000000000000000000'
+
+const logger = new Logger(module)
+
+export class VoteOnSuspectNodeHelper {
+    provider: Provider
+    address: string
+    contract: Operator
+    signer: Signer
+    streamIdOfSponsorship: Map<string, string> = new Map()
+    sponsorshipCountOfStream: Map<string, number> = new Map()
+    callback: (sponsorship: string, operatorContractAddress: string) => void
+
+    constructor(config: OperatorServiceConfig,
+        callback: (sponsorship: string, soperatorContractAddress: string) => void) {
+        logger.trace('OperatorClient created')
+        this.callback = callback
+        this.address = config.operatorContractAddress
+        this.provider = config.provider
+        this.signer = config.signer
+        this.signer.connect(this.provider)
+        this.contract = new Contract(config.operatorContractAddress, operatorABI, this.signer) as unknown as Operator
```

Comment:
I believe field `signer` can be made into a local const.
```suggestion
        const signer  = config.signer
        signer.connect(this.provider)
        this.contract = new Contract(config.operatorContractAddress, operatorABI, signer) as unknown as Operator
```